bin/addEntrypoint.bash

```diff
@@ -65,17 +74,13 @@ if [[ ! "$FEATURE_BRANCH_EXISTS" == "" ]]; then
   exit 255
 fi
 
-# Check if the directory for the entry point exits...
+# Check if the directory for the entry point exists...
 if [ -d "src/entrypoints/$ENTRY_POINT" ]; then
   echo -e "${ON_RED}ERROR:${NO_COLOR} A directory for the entry point $ENTRY_POINT already exists"
-  echo "in the directory $$ENTRY_POINT_SRC_DIR."
+  echo "in the directory $ENTRY_POINT_SRC_DIR."
   echo "Pick a different name for your entry point."
   echo "OR:"
```

Comment:
Probably clarify here:  "OR if you are sure you want to replace the entry point:"